UnitedIncome/components
# Adds react-hooks eslint plugin

### Description
> Please provide a brief description of your changes below.

Adds the react-hooks eslint rules. 

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Nothing to test really. It already passes.
